distribution/packages/src/deb/debian/postinst

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Copyright Wazuh Indexer Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The Wazuh Indexer Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+# deb wazuh-indexer postinst script
+
+set -e
+
+echo "Running Wazuh Indexer Post-Installation Script"
+
+product_dir=/usr/share/wazuh-indexer
+config_dir=/etc/wazuh-indexer
+data_dir=/var/lib/wazuh-indexer
+log_dir=/var/log/wazuh-indexer
+pid_dir=/var/run/wazuh-indexer
+
+# Apply Security Settings
+if [ -d ${product_dir}/plugins/opensearch-security ]; then
+    bash ${product_dir}/plugins/opensearch-security/tools/install_demo_configuration.sh -y -i -s > ${log_dir}/install_demo_configuration.log 2>&1
+fi
+
+# Apply PerformanceAnalyzer Settings
+chmod a+rw /tmp
+if ! grep -q '## Wazuh Indexer Performance Analyzer' ${config_dir}/jvm.options; then
+   # Add Performance Analyzer settings in ${config_dir}/jvm.options
+   CLK_TCK=`/usr/bin/getconf CLK_TCK`
+   echo >> ${config_dir}/jvm.options
+   echo '## Wazuh Indexer Performance Analyzer' >> ${config_dir}/jvm.options
+   echo "-Dclk.tck=$CLK_TCK" >> ${config_dir}/jvm.options
+   echo "-Djdk.attach.allowAttachSelf=true" >> ${config_dir}/jvm.options
+   echo "-Djava.security.policy=file://${config_dir}/opensearch-performance-analyzer/opensearch_security.policy" >> ${config_dir}/jvm.options
+   echo "--add-opens=jdk.attach/sun.tools.attach=ALL-UNNAMED" >> ${config_dir}/jvm.options
+fi
```

Comment:
We need to check if these settings exist already in `jvm.prod.options`. If they do, we need to decide whether to keep them.